test/integration/api/external/index.js

```diff
@@ -0,0 +1,245 @@
+// describes the API used by axe Pro
+describe('external API', function() {
+	'use strict';
+
+	afterEach(function() {
+		// setup _tree as needed, always reset
+		axe._tree = null;
+	});
+
+	describe('axe.commons.text.sanitize', function() {
+		it('typeof fn', function() {
+			assert.isFunction(axe.commons.text.sanitize);
```

Comment:
Is this test actually useful?

Reply:
I don't think we need this test, as the test below it requires `axe.commons.sanitize` to be a function in order to pass.